_includes/snippets_library/Birmingham_Baskerville_slurm/_config_options.yml

```
@@ -0,0 +1,68 @@
+#------------------------------------------------------------
+# Birmingham Baskerville  Slurm: Jenny Wong
+#------------------------------------------------------------
+
+# Cluster host and scheduler options: the defaults come from
+# Graham at Compute Canada, running Slurm. Other options can
+# be found in the library of snippets,
+# `_includes/snippets_library`. To use one, replace options
+# below with those in `_config_options.yml` from the
+# library. E.g, to customise for Cirrus at EPCC, running
+# PBS, we could replace the options below with those from
+# 
+# _includes/snippets_library/EPCC_Cirrus_pbs/_config_options.yml
+# 
+# If your cluster is not represented in the library, please
+# copy an existing folder, rename it, and customize for your
+# installation. Remember to keep the leading slash on the
+# `snippets` variable below!
+
+snippets: "/snippets_library/Birmingham_Baskerville_slurm"
+
+local:
+  prompt: "[user@laptop ~]$"
+  bash_shebang: "#!/usr/bin/env bash"
+
+remote:
+  name: "Baskerville"
+  login: "login.baskerville.ac.uk"
+  host: "bask-pg0310u18a.cluster.baskerville.ac.uk"
+  node: "bask-pg"
+  location: "University of Birmingham, UK"
+  homedir: "/bask/homes/y/yourUsername"
```

```suggestion
  homedir: "/bask/homes/y/"
```

Get rid of extra username subdir in path